readme.md

```diff
@@ -9,44 +9,67 @@ Hosted version is available at https://yearn.vision.
 
 You will need:
 
-- Erigon for querying historical data
-- Victoria-metrics to pull the metrics, persist them and make them queryable
-- Grafana if you want to set up custom dashboards and alerts
-- Etherscan API key
-- docker and docker-compose (not mandatory but easier usage, see below)
+- Etherscan API key (and API keys for other networks that you want to use)
```

Comment:
we shoudl clarify that these are block explorer api keys

Reply:
```suggestion
- Etherscan API key (and API keys for other networks block explorer that you want to use)
```